tests/std/tests/Dev11_0535636_functional_overhaul/test.cpp

```diff
@@ -1049,8 +1049,7 @@ void test_reference_wrapper_invocation() {
 }
 
 
-// Test C++17 invoke().
-#if _HAS_CXX17
+// Test invoke().
 constexpr bool test_invoke_constexpr() {
     // MSVC implements LWG-2894 in C++17 and later

```

Comment:
This comment should be updated.

Additionally, `<yvals_core.h>` has comments that need to be updated.